WalletWasabi/Services/WalletService.cs

```diff
@@ -383,25 +371,13 @@ private void ProcessBlock(Height height, Block block, IEnumerable<int> filterByT
 			{
 				if (filterByTxIndexes is null)
```

Comment:
This has to be removed because we don't have the indexes anymore.

Reply:
Done.